incompatibility/reporter.go

```diff
@@ -143,6 +143,9 @@ func ComponentsSearch(doc *openapiv3.Document) []*Incompatibility {
 			)
 		}
 	}
+	if doc.Components.SecuritySchemes != nil {
```

Comment:
I don't see a test for this new case.

Reply:
I saw your message from the other PR that you now have a spec file which has this.  But we aren't actually testing that this case is found.    See the comment I left in incomp-scanning_tests.go.  Let me know if you have questions or want to chat.

Reply:
leaving ticket to indicate future testing is need see [here](https://b.corp.google.com/issues/195057972)

Reply:
ack